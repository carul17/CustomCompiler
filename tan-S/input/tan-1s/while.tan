main {
	var a := [1,2,3];
	var b := 0;
	while(b < 3){
		print [a:b] \n;
		b := b-1;
	}
	
}
