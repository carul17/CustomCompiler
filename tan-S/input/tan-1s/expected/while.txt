[ 129 ]
[ 5 1 ]
[ 5 ]
5
[ 129 129 ]
[ 129 5 1 ]
[ 129 ]
[ 4 1 ]
[ 4 ]
4
[ 129 129 ]
[ 129 4 1 ]
[ 129 ]
[ 3 1 ]
[ 3 ]
3
[ 129 129 ]
[ 129 3 1 ]
[ 129 ]
[ 2 1 ]
[ 2 ]
2
[ 129 129 ]
[ 129 2 1 ]
[ 129 ]
[ 1 1 ]
[ 1 ]
1
[ 129 129 ]
[ 129 1 1 ]
[ 129 ]
[ 0 1 ]
